framework/test_app/runners/k8s/k8s_base_runner.py

```diff
@@ -196,6 +196,7 @@ def run(self, **kwargs):
             self.namespace = self._create_namespace(
                 self.namespace_template, namespace_name=self.k8s_namespace.name
             )
+            self.k8s_namespace.wait_for_namespace_active()
```

Comment:
We should be leave a log message when we're waiting for something so it doesn't look like the driver is just stuck doing nothing

```suggestion
            logger.info(
                "Waiting for namespace %s to become active",
                self.k8s_namespace.name,
            )
            self.k8s_namespace.wait_for_namespace_active()
```